python/lsst/sdm/schemas/apdb.yaml

```diff
@@ -1481,8 +1024,8 @@ tables:
     datatype: boolean
     description: Aperture did not fit within measurement image.
     fits:tunit:
-  - name: is_negative
-    "@id": "#DiaSource.is_negative"
+  - name: isNegative
+    "@id": "#DiaSource.isNegative"
```

Comment:
One big caveat - Cassandra does not support renaming of the (non-PK) columns. When doing schema migration for existing database I can drop "is_negative" column and create a new one, but it will mean all "is_negative" data is lost. If this is not acceptable then the only option is to stick with the existing name. We could potentially add an option for column renaming on client side (when writing and reading the data) but that will need some development.

Reply:
OK, I'll drop and create a new column.